src/main/resources/view/DarkTheme.css

```diff
@@ -100,7 +100,7 @@
 }
 
 .list-cell:filled:even {
-    -fx-background-color: #3c3e3f;
+    -fx-background-color: transparent;
```

Comment:
I think the problem still remains because there is still list-cell:filled:odd. But you can't change list-cell:filled:odd to transparent too, otherwise u will affect the persons and pet patients card UI. Would it be possible to set a custom styling for calendar? e.g. set the text of the calendar (affected area) to white

Reply:
![image](https://user-images.githubusercontent.com/15257617/38463931-0fe41e28-3b37-11e8-905b-148180ddb6bb.png)